dockerfiles/ubuntu/is-analytics/Dockerfile

```diff
@@ -18,29 +18,34 @@
 
 # set to latest Ubuntu LTS
 FROM ubuntu:16.04
-MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
+      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.6.0.2"
 
 # set user configurations
 ARG USER=wso2carbon
 ARG USER_ID=802
 ARG USER_GROUP=wso2
 ARG USER_GROUP_ID=802
 ARG USER_HOME=/home/${USER}
-# set dependant files directory
-ARG FILES=./files
-# set jdk configurations
-ARG JDK=jdk1.8.0*
+# set JDK configurations
 ARG JAVA_HOME=${USER_HOME}/java
+ARG JDK_NAME=jdk-8u251-linux-x64.tar.gz
+ARG JDK_URL
 # set wso2 product configurations
 ARG WSO2_SERVER=wso2is-analytics
 ARG WSO2_SERVER_VERSION=5.6.0
 ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_DIST_URL
+# mysql connector
+ARG MYSQL_CONNECTOR_VERSION=5.1.49
 
 # install required packages
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     curl \
+    unzip \
+    wget \
```

Comment:
@NishikaDeSilva shall we maintain the package names in the alphabetical order? Please refer to official [documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments).